tasks/db_deploy/test/manual_tests/README.md

```diff
@@ -613,6 +625,11 @@ PostgreSQL *disaster_recovery* database. Perform the checks below by going to th
     orca   | recovery_file   | table | orca_dbo
     orca   | recovery_job    | table | orca_dbo
     orca   | recovery_status | table | orca_dbo
+    orca   | providers       | table | orca_dbo
```

Comment:
The new tables in the list (i.e. collections, providers, etc.) should be replicated around line 326 when the tables are displayed during a test.

Also around line 329, the version information from the schema_version table should be updated to reflect the proper version that should be displayed.

Reply:
updated